platform/core/src/utils/formatPN.js

```diff
@@ -6,7 +6,13 @@ export default function formatPN(name) {
     return;
   }
 
-  const nameToUse = name.Alphabetic ?? name;
+  let nameToUse;
+  const _nameToUse = name.Alphabetic ?? name;
+  if (_nameToUse instanceof String) {
```

Comment:
Ok. I just changed that because for some reason there is an entry in the IDC dicomweb server that returns an object ( {} ) in the name and prevents OHIF to display the study list